GPU/GPUTracking/Global/GPUChainTrackingTRD.cxx

```diff
@@ -38,11 +38,10 @@ int GPUChainTracking::RunTRDTracking()
   mRec->PushNonPersistentMemory(qStr2Tag("TRDTRACK"));
   SetupGPUProcessor(&Tracker, true);
 
-  for (unsigned int iTracklet = 0; iTracklet < mIOPtrs.nTRDTracklets; ++iTracklet) {
-    if (Tracker.LoadTracklet(mIOPtrs.trdTracklets[iTracklet], mIOPtrs.trdTrackletsMC ? mIOPtrs.trdTrackletsMC[iTracklet].mLabel : nullptr)) {
-      return 1;
-    }
+  if (mIOPtrs.trdTracklets == 0) {
+    return 0;
   }
+  // TODO what about trdTrackletsMC? Are they needed at all?
```

Comment:
That already exists in the IOPtr, so you can just fill it from external as you do for the tracklet itself

Reply:
Yes, but that was originally added for AliRoot, but in the end I never used it. For o2 the labels are in a separate container anyways. So I think I will remove that entirely

Reply:
ok, makes sense